hl7-mllp/2.1/modules/ROOT/edits/hl7-mllp-connector-examples.adoc

```diff
@@ -0,0 +1,136 @@
+= HL7 EDI - Examples - Mule 4
+
+Use these examples to read and write HL7 messages over the MLLP protocol.
+
+== Example: HL7 Studio
+
+Set these Global properties before using the examples:
+
+* Delimiter: The standard HL7 delimiter value that separates
+MLLP messages. It is a hexadecimal 0x1C character also known as FS. Applications in some clinics use a different delimiter value, that can be set in this field. The standard value in the
+HL7 MLLP connector is shown as `001C`.
++
+* Host: Sets the socket host address.
+* Port: Sets the socket port.
++
+For an advanced connection, complete these fields:
++
+* Reuse address: Whether to reuse the configured socket or fail when trying to bind to the socket.
+* Send TCP With No Delay: Whether or not to transmit data immediately, or collect data for greater efficiency before sending.
+* Linger: How long in milliseconds that the socket takes to close so any remaining data transmits correctly.
```

Comment:
```suggestion
* Linger
+
How long, in milliseconds, the socket can take to close without preventing remaining data from transmitting correctly.
```
Please verify accuracy of rewording; the original wording is awkward (how long...so data transmits...)

Reply:
fixed